qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/QueryANNIT.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
nit: Did you generate this license or was it added by IDE? we moved to shorter version, if IDE added longer version, then we might need to fix this for another commit.

Reply:
I believe I just copy pasted it from somewhere, is there a preference I don't mind changing it

Reply:
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
 ```